addons/website/views/snippets/snippets.xml

```diff
@@ -954,6 +954,51 @@
             data-page-option-name="header_text_color"/>
     </div>
 
+    <div data-js="HeaderExtraElts"
+         data-selector="#wrapwrap > header"
+         data-no-check="true"
+         groups="website.group_website_designer">
+        <we-row string="Elements" class="o_we_full_row align-items-start">
+            <div class="w-100">
+                <div class="d-flex gap-1 mb-1">
```

Comment:
I am pretty sure we don't need that it's suppose to be layed out automatically. Can't you just make multiple we-row? Would that not work? If not, it's alright, we'll review with OWL later anyway :wink:

Reply:
I updated it using double `<we-row/>` :)